prog/dagorInclude/util/dag_bindump_ext.h

```diff
@@ -439,6 +439,20 @@ class string : public eastl::string
 
 public:
   BINDUMP_ENABLE_STREAMING(string, eastl::string, string);
+
+  // GCC complains of ambiguous overload if we don't specify an
+  // operator overload for C strings. Might benefit Clang and
+  // MSVC if we just specify the overload as well.
+  string &operator=(const char* other)
+  {
+    eastl::string::operator=(other);
```

Comment:
Maybe try to do `using eastl::string::operator=;` instead of implementing it manually? Not sure whether GCC will like that, but worth a shot.